packages/charts/test/exporting.test.js

```diff
@@ -131,6 +131,37 @@ describe('vaadin-chart exporting', () => {
     expect(styleRemovedFromBody).to.be.true;
   });
 
+  it('should add styled-mode attribute to body before export and delete it afterwards', async () => {
```

Comment:
Let's add another test to verify that `styled-mode` attribute is NOT set on the `body` when `styledMode` config option is set to `false` (see my previous comment).

Reply:
Addressed with the latest commits